daprdocs/content/en/reference/api/jobs_api.md

```diff
@@ -0,0 +1,146 @@
+---
+type: docs
+title: "Jobs API reference"
+linkTitle: "Jobs API"
+description: "Detailed documentation on the jobs API"
+weight: 1300
+---
+
+{{% alert title="Note" color="primary" %}}
+The jobs API is currently in alpha.
```

Comment:
Can you also update this table https://docs.dapr.io/operations/support/alpha-beta-apis/ for the jobs alpha API